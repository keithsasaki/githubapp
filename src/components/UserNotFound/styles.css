#user-not-found {
    margin: 0 auto;
    font-size: 4.2rem;
    color: var(--color-purple);
}
