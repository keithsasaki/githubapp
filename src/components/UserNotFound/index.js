import React from 'react';

import './styles.css';

function UserNotFound(){
    return(
        <div id="user-not-found">
            <span>User not found :(</span>
        </div>
    );
}

export default UserNotFound;