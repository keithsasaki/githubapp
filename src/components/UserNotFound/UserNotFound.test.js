import React from 'react';
import '@testing-library/jest-dom';

import renderer from 'react-test-renderer';
import UserNotFound from '.';

describe("Testing UserNotFound Component", () => {
    it("matches snapshot", () => {
        const tree = renderer.create(<UserNotFound />).toJSON();
        expect(tree).toMatchSnapshot();
    });
})