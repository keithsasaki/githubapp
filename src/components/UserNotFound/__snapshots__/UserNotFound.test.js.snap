// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing UserNotFound Component matches snapshot 1`] = `
<div
  id="user-not-found"
>
  <span>
    User not found :(
  </span>
</div>
`;
